### PR TITLE
CON-160 Enable admin to update notification preferences

### PR DESCRIPTION
**Pull request by mwinel**
### Description
This PR adds the capability for the admin to turn their notifications off and on to switch their ability to view notifications in the preference page.
### Type of change
This is a new feature.
### How Has This Been Tested?

- Clone the repo: git clone https://github.com/andela/mrm_api.git.
- Setup the project as per the ` README.md `.
- Checkout to branch ` story/CON-160-update-admin-notification-preference `.
- Run the following query to trigger notification settings on ` getNotifications `.
```
mutation {
  updateNotification(
    deviceHealthNotification: true,
    meetingUpdateNotification: true,
    getNotifications: false) {
    notification {
      id
      userId
      deviceHealthNotification
      meetingUpdateNotification
      getNotifications
    }
  }
}
```
- Run the following query to get all notifications.
```
query{
  allUnreadNotifications{
    notifications{
      id
      title
      message
    }
  }
}
```
### JIRA
[CON-160](https://andela-apprenticeship.atlassian.net/browse/CON-160)

### CODE REVIEW

**Comment by joshuaocero** [comment]
This PR naming does not follow proper engineering conventions. Please refer to the Engineering Playbook.

**Comment by IssaIan** [comment]
@mwinel Please update the PR name accordingly. The story id should come first.

**Comment by IssaIan** [comment]
@mwinel after rebasing, you need to update your alembic revisions. This is the reason your tests are currently failing. We can have a sync on this if you aren't conversant with how to go about it.

File: alembic/versions/79ef610dbd41_add_activity_column_to_devices_table.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+"""add activity column to devices table
+
+Revision ID: 79ef610dbd41
+Revises: af8e4f84b552
```
Comment (IssaIan) [inline alembic/versions/79ef610dbd41_add_activity_column_to_devices_table.py]: Check the alembic revisions revisions as explained on the comment.

**Comment by rajeman** [comment]
Tasks related to admin notifications have been suspended.

**Pull request closed without merging**